pyproject.toml

```diff
@@ -81,20 +74,7 @@ optional = true
 ruff = "^0.9.1"
 
 [tool.poetry.extras]
-qblox = [
-  "qblox-instruments",
-  "qcodes",
-  "qcodes_contrib_drivers",
-  "pyvisa-py",
-  "lark",
-]
-qm = ["qm-qua"]
-zh = ["laboneq"]
-rfsoc = ["qibosoq"]
-los = ["qcodes", "qcodes_contrib_drivers", "pyvisa-py"]
-twpa = ["qcodes", "qcodes_contrib_drivers", "pyvisa-py"]
 emulator = ["qutip"]
-bluefors = ["pyyaml"]
 qrng = ["pyserial"]
```

Comment:
Is this tested in the CI?

Reply:
I think yes, in https://github.com/qiboteam/qibolab/blob/main/tests/instruments/test_qrng.py